lib/parsers/entity_parser.js

```diff
@@ -107,7 +107,16 @@ function resetState() {
   isType = null;
 }
 
-function initializeEntities() {
+function initializeEntities(args) {
+  let creationTimestamp;
+  if (args.creationTimestamp) {
```

Comment:
here we're not really initializing entities: we're setting up the creation timestamp
it should belong to another function